segments/tests/test_tokenizer.py

```diff
@@ -75,7 +75,7 @@ def test_ipa(self):
         self.assertEqual(t('\u02b0ello', ipa=True), '\u02b0e l l o')
 
     def test_tokenize_with_profile(self):
-        self.assertEqual(self.t('aa'), 'b')
+        self.assertEqual(self.t('aa'), ' b')
```

Comment:
The rules file contains whitespace:
```
a{2}, b
```
which was stripped before. But I think the better behaviour is not to strip, e.g. to allow actually including whitespace that way.